src/content/docs/ja/reference/gritql.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: GritQL [実験的機能]
+description: Biome における GritQL の基本的な使い方
+---
+
+GritQL は、ソースコードの構造検索を実行するためのクエリ言語です。
+つまり、空白や、文字列で使用される引用符の種類などの些細な情報は、検索クエリでは無視されます。
+さらに、スニペット、マッチング、ネスト、変数など、構文構造をクエリできる多くの機能も提供します。
+
+GritQL は[オープンソース](https://github.com/getgrit/gritql/)であり、[Grit.io](https://grit.io/)によって作成されました。
+
+Biome は GritQL を 2 つの目的で統合しています：
+
+- IDE 拡張機能にも拡張したいと考えている、[`biome search`](/reference/cli/#biome-search) コマンド
+- 現在進行中の、プラグインへの取り組み
+
+## パターン
+
+GritQL クエリは _パターン_ を通じて機能します。
+最もよく見られるパターンは、バッククォートで囲まれた通常のソースコードのように見えるコードスニペットです。
+
+```grit
+`console.log('Hello, world!')`
+```
+
+このパターンは、文字列 `'Hello, world!'` が渡された全ての `console.log()` の呼び出しに一致します。
+ただし、GridQLは _構造的_ マッチングを行うため、フォーマットの詳細には関心がありません。
```

Comment:
textlint says we prefer to use「format」 to 「フォーマット」, but I think it is not proper for this sentence. Is there any idea?

Reply:
yeah, I think we should use リント and フォーマット instead of alphabet in overall now. (Initially I thought that リント is a bit awkward, so I decided to use lint and format as alphabet because they are also technical words.)

Could you fix it to pass textlint ci for now? I will adjust everything later.

Reply:
I understand, thank you!
I fixed temporarily it to be "format"!